src/main/java/hudson/plugins/brakeman/scanners/BrakemanTabsScanner.java

```diff
@@ -0,0 +1,60 @@
+package hudson.plugins.brakeman.scanners;
+
+import hudson.plugins.analysis.core.ParserResult;
+import hudson.plugins.analysis.util.model.Priority;
+import hudson.plugins.brakeman.Warning;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * A Java class that represents a tabs Scanner of the Brakeman Output file
+ * This class includes a specific strategy for implementing the parsing process
+ * of the output file.
+ *
+ * @author Thomas Baird
+ */
+public class BrakemanTabsScanner implements AbstractBrakemanScanner {
+
+    private static Pattern pattern = Pattern.compile("^([^\t]+?)\t(\\d+)\t([\\w\\s]+?)\t(\\w+)\t([^\t]+?)\t(High|Medium|Weak)", Pattern.MULTILINE);
+    /**
+     * Creates a new instance of <code>BrakemanTabsScanner</code>
+     */
+    public BrakemanTabsScanner() {}
+
+    public void scan(String content, ParserResult project) {
+        Matcher m = pattern.matcher(content);
+
+        this.scanWarnings(m, project);
```

Comment:
i love the use of `this.` when referring to instance variables because it makes it super duper clear.  however, i don't think that you need to do the same with instance methods ... it has been a while since i have written java and neither the soracle or google style guides cover it ... so my recommendation it to follow the convention of the existing code base.